weechat-android/src/main/java/com/ubergeek42/WeechatAndroid/service/BootUpReceiver.java

```diff
@@ -13,8 +13,9 @@
     @Override
     public void onReceive(Context context, Intent intent) {
         SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
-        if (prefs.getBoolean(PREF_AUTO_START, PREF_AUTO_START_D)) {
+        if (prefs.getBoolean(PREF_BOOT_CONNECT, PREF_BOOT_CONNECT_D)) {
```

Comment:
Given the abundance of preference changes, we may want to pop up a notice for upgrading users that many preferences have changed. Or, provide an upgrade path which will migrate preferences. Users will expect things to work as they did previously, or at least know that they won't.